azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/internal/BaseAzureComputeApiLiveTest.java

```diff
@@ -74,6 +83,17 @@ protected String getEndpoint() {
       return endpoint;
    }
 
+   protected String getSubscriptionId() {
+      String subscriptionid = null;
+      String endpoint = null;
+      endpoint = getEndpoint();
+      if (endpoint != null) {
+         subscriptionid = endpoint.substring(endpoint.lastIndexOf("/") + 1);
+      }
+      assertNotNull(subscriptionid);
+      return subscriptionid;
+   }
```

Comment:
@nacx `getSubscriptionId` is needed but only for two live tests, not in the actual provider code. We don't actually need `subscriptionId` to be injected anywhere in the provider code. Note the above payload no longer needs it. Do we have inject this value?